wordpress-mobile/WordPress-Android
# Reader: Updates search related analytics.

Updates analytics related to reader search.

To test:
Set break points where the relevant stats are bumped.
Test the app and confirm that the stats are bumped as expected and there are no warnings/crashes.

Needs review: @nbradbury if you please sir
